app/components/dashboard/DcChart.vue

```
@@ -3883,18 +3765,39 @@ const mm = {
             'name_filter',
             'name2',
             'name2_filter',
+            'name2_order',
```

ESLint

[error] Expected indentation of 10 spaces but found 12.

(@stylistic/indent)